atomics/T1115/T1115.yaml

```diff
@@ -26,4 +26,13 @@ atomic_tests:
       echo Get-Process | clip
       Get-Clipboard | iex
     name: powershell
+- name: Execute commands from clipboard
+  description: Echo a command to clipboard and execute it
+  supported_platforms:
+  - macos
+  executor:
+    command: |-
+      echo ifconfig | pbcopy
+      $(pbpaste)
+    name: sh
```

Comment:
Call out bash here, since the command substitution you're using is bourned based (or zsh)